packages/lib/OgImages.tsx

```diff
@@ -127,7 +127,6 @@ export const Meeting = ({ title, users = [], profile }: MeetingImageProps) => {
   const avatars = attendees
     .map((user) => {
       if ("image" in user && user?.image) return user.image;
-      if ("username" in user && user?.username) return `${CAL_URL}/${user.username}/avatar.png`;
```

Comment:
Do we really need this line? I couldn't find relevant code for storing users' avatars in `${user.username}/avatar.png`.